requirements.txt

```diff
@@ -1,2 +1,3 @@
 torch>=1.5
 numpy>=1.17
+cupy
```

Comment:
Please remove this dependency from here. We will add it to the images with CUDA later.